mmv1/third_party/terraform/services/compute/resource_compute_router_peer.go.erb

```diff
@@ -1054,6 +1140,21 @@ func flattenNestedComputeRouterBgpPeerEnableIpv6(v interface{}, d *schema.Resour
 	return v
 }
 
+<% unless version == 'ga' -%>
+func flattenNestedComputeRouterBgpPeerEnableIpv4(v interface{}, d *schema.ResourceData, config *transport_tpg.Config) interface{} {
+	log.Printf("[DEBUG] printing value for enable_ipv4 that we are going to set %v)", v)
```

Comment:
This log can be deleted? It looks like it is for developing.

Reply:
Done